doc/api/policy.md

```diff
@@ -0,0 +1,103 @@
+# Policies
+
+<!--introduced_in=TODO-->
+<!-- type=misc -->
+
+> Stability: 1 - Experimental
+
+<!-- name=policy -->
+
+Node.js contains experimental support for creating policies on loading code.
+
+Policies are a security feature intended to allow guarantees
+about what code Node.js is able to load. The use of policies assumes
+safe practices for the policy files such as ensuring that policy
+files cannot be overwritten by the Node.js application by using
+file permissions.
+
+A best practice would be to ensure that the policy manifest is read only for
+the running Node.js application, and that the file cannot be changed
+by the running Node.js application in any way. A typical setup would be to
+create the policy file as a different user id than the one running Node.js
+and granting read permissions to the user id running Node.js.
```

Comment:
In the current design, this is only recommended, not enforced anyhow, or otherwise encouraged through warnings?

Reply:
It is prohibitively hard to do this check properly. We should not try to enforce it unless someone can come up with a solid solution. We could warn, but it also would be hard to detect all possible cases that could allow mutation. Sometimes the mutation is even desirable if you are editing a policy file such as prompting on error.